pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -992,7 +991,13 @@ func TestDryRunPreemption(t *testing.T) {
 				st.RegisterBindPlugin(defaultbinder.Name, defaultbinder.New),
 			)
 			registeredPlugins = append(registeredPlugins, tt.registerPlugins...)
-			objs := []runtime.Object{&v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: ""}}}
+			var objs []runtime.Object
+			for _, p := range append(tt.testPods, tt.initPods...) {
+				objs = append(objs, p)
+			}
+			for _, n := range nodes {
+				objs = append(objs, n)
+			}
```

Comment:
can we do something like the following?
```
objs := append([]runtime.Object{}, tt.testPods...)
objs = append(objs, tt.initPods...)
objs = append(objs, nodes...)

```

Reply:
Unfortunately not - Go doesn't support generic type yet :)